slidez-fe-core/src/common/components/forms/LoginForm.tsx

```diff
@@ -106,13 +108,14 @@ const LoginForm = ({ onLogin, onLoginWithGoogle }: LoginProps) => {
                     />
                 </div>
             </div>
-            <div className='form-row buttons-row'>
-                <button
+            <div className='form-row' />
+            <div className='form-row'>
+                <GeneralButton
                     className='form-button login-button'
                     onClick={handleLogin}
                 >
                     Log In
```

Comment:
Can you add text to the button definition? As mostly it will have only text. And if it's empty, then set children